admin_scripts/ubuntu/protect_settings.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -ex
+
+if [ !  -f "/usr/bin/gnome-control-center.real" ]
+then
+    cp  /usr/bin/gnome-control-center /usr/bin/gnome-control-center.real
+fi
+
+
+cat << EOF > /usr/bin/gnome-control-center 
+#!/bin/bash
+
+USER=\$(id -un)
+
+if [ \$USER == "user" ]; then
+  zenity --info --text="Systemindstillingerne er ikke tilgængelige for publikum.\n\n Kontakt personalet, hvis der er problemer."
+else
+  /usr/bin/gnome-control-center.real
+fi
+
+EOF
+
+chmod +x /usr/bin/gnome-control-center
```

Comment:
I would suggest that we also do `chown root:superuser superuser /usr/bin/gnome-control-center.real && chmod 770 /usr/bin/gnome-control-center.real` at some point: we want `superuser` and `root` to be able to run it, but the normal user shouldn't be able to _read_ it, let alone execute it.

Reply:
(In the light of my other comment, I'd like to suggest using `dpkg-statoverride` for this purpose rather than `chmod`.)